examples/flutter_gallery/.idea/workspace.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We don't want to commit the `workspace.xml` files - these are meant to have per-user settings (window locations, ...).